src/pystatis/table.py

```diff
@@ -154,7 +154,9 @@ def get_data(
             decimal="," if language == "de" else ".",
             dtype={raw_data_header.split(";")[pos + 2]: str for pos in pos_of_ags_col},
             parse_dates=[config.LANG_TO_COL_MAPPING[db_version][language]["time"]],
-            date_format="%d.%m.%Y" if language == "de" else "%Y-%m-%d",
+            date_format="%d.%m.%Y"
+            if (language == "de" and db_name != "zensus")
```

Comment:
are you sure this is still correct? Please check with current Genesis version as they switch to the new database version that is like Zensus format so I would actually expect they have the same format now?

Reply:
Fixed this so genesis and zensus now both use the new format for either language. Added tests for these.